tests/integration/acp/query/avg_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package test_acp
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+	acpUtils "github.com/sourcenetwork/defradb/tests/integration/acp"
+)
+
+func TestACP_QueryAverageWithoutIdentity(t *testing.T) {
```

Comment:
todo: I see no good reason why this is a single test function.  It tests two totally separate things, on two different collections.  There isn't even a join between them in this test(s).

It will require the maintainer to manually comment out different test actions in order to debug.  Please dont do this.

The same goes for the other tests.

Reply:
agreed. Wanted to do it later, but forgot. Thanks for pointing out.